### PR TITLE
[IMP] inventory: update 2 step routes doc

### PR DESCRIPTION
**Pull request by brse-odoo**
This PR is an [IMP] to the `receipts_delivery_two_steps` doc in Odoo's Inventory documentation, specific to 17.0.

Here's what I did:
- Retooled some of the instructions/flow for 17.2 update that affected all multi-step routes
- Replaced all screenshots for 17.2 UI, removed some screenshots no longer relevant

**Note:** _This PR was only to update small instructions and replace screenshots. A second, targeted PR for a more robust rewrite will follow once this is merged._

[Project Task Link](https://www.odoo.com/web#id=3878113&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

### CODE REVIEW

**Comment by brse-odoo** [comment]
Hi @odoo/inventory-doc-review, this PR is ready for review when one of you has a chance. All that needs to be reviewed are the few lines that were changed, and the replaced screenshots. Thank you!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (hojo-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:83]:
```suggestion
After confirming the :abbr:`PO (Purchase Order)`, a :guilabel:`Receipt` smart button appears at
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (hojo-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:100]:
```suggestion
to complete the receipt and move the product to :guilabel:`WH/Input`.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (hojo-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:120]:
```suggestion
Once the transfer is validated, the product enters inventory and is available for customer
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (hojo-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:180]:
```suggestion
The :guilabel:`Delivery` smart button now indicates there are two transfers. Click the
```

**Review by hojo-odoo — APPROVED** [review]
Hey @brse-odoo,

Nice job with these updates. I'm approving with a few small suggestions.

Thanks,
hojo

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:83]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:100]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:120]: Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:180]: Suggestion implemented ✅

**Comment by brse-odoo** [comment]
Hi @odoo/us-doc-review, this PR is ready for final content review. Thank you!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:42]: Not against it...just don't see the change here other than adding 10kb.

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:101]: Too zoomed in, would include the larger form/UI

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:140]: Too zoomed in, would include the larger form/UI

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:186]: Too zoomed in, would include the larger form/UI

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:122]: Too zoomed in, would include the larger form/UI

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:167]: Too zoomed in, would include the larger form/UI

**Review by StraubCreative — APPROVED** [review]
Hi @brse-odoo 
Approving with similar comments to the previous PRs.
Can pass to SALI when done, thanks!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:101]: Replaced with new, zoomed-out screenshot that showcases navigational UI and shows key areas. Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:140]: Replaced with new, zoomed-out screenshot that showcases navigational UI and shows key areas. Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:186]: Replaced with new, zoomed-out screenshot that showcases navigational UI and shows key areas. Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:167]: Replaced with new, zoomed-out screenshot that showcases navigational UI and shows key areas. Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:42]: The old screenshot was in a 15.0 db, and was replaced with a screenshot from a 17.2 database. It does look similar, but there are very small differences - the text is slightly thinner and less bold

**Comment by brse-odoo** [comment]
> Hi @brse-odoo Approving with similar comments to the previous PRs. Can pass to SALI when done, thanks!

Hi @samueljlieber, this PR has passed us-doc-review and is ready for your final lookover before merge, thank you!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst
Comment (samueljlieber) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst:147]: Small fix :)
```suggestion
   :guilabel:`Pick` Kanban card.
```

**Review by samueljlieber — APPROVED** [review]
Hi @brse-odoo, nice job on the improvements to this doc! Your changes look good to me, approving and delegating to you with one quick fix.
..
@robodoo delegate=brse-odoo

**Comment by brse-odoo** [comment]
> Hi @brse-odoo, nice job on the improvements to this doc! Your changes look good to me, approving and delegating to you with one quick fix. .. @robodoo delegate=brse-odoo

@robodoo r+

**Pull request closed without merging**
